ui/lib/kv/addon/utils/kv-breadcrumbs.js

```diff
@@ -45,7 +45,7 @@ export function breadcrumbsForSecret(backend, secretPath, lastItemCurrent = fals
         };
       }
       if (!isDir) {
-        return { label: segment.label, route: 'secret.details', models: [backend, segment.model] };
+        return { label: segment.label, route: 'secret', models: [backend, segment.model] };
```

Comment:
We might have missed `secret.index` here?
```suggestion
        return { label: segment.label, route: 'secret.index', models: [backend, segment.model] };
```

Reply:
Oh yeah - I didn't change this one intentionally because I was getting strange failures but let me look back into